modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js

```diff
@@ -1629,6 +1629,18 @@ AUI.add(
 						'#' + instance.getInputName() + 'Title'
 					);
 
+					var parsedAssetTitleMap = instance.getParsedValue(
+						parsedValue.assetTitleMap
```

Comment:
This one should be `parsedValue.assetTitle`

Reply:
I have renamed the former JSON attribute assetTitle to assetTitleMap in line 1809 so it should be parsedValue.assetTitleMap as is.

Reply:
Sorry, I had the old code in 1809 cached in the browser for some reason, even after I clean deployed. You are right, it's working as it is! :)

Reply:
No problem :)